examples/gallery/symbols/multi_parameter_symbols.py

```diff
@@ -36,15 +36,25 @@
 fig.plot(data=data, style="j", color="red3", pen="2p,black")
 
 # RECTANGLE
-data = [[3, 1, 4, 1.5]]
+data = [[3, 0.5, 4, 1.5]]
 fig.plot(data=data, style="r", color="dodgerblue", pen="2p,black")
 
+# RECTANGLE
+# width/height are given via the style parameter
+data = [[3, 1.5]]
+fig.plot(data=data, style="r4/1.5", color="dodgerblue", pen="2p,black")
```

Comment:
Would using x/y be better?

```suggestion
fig.plot(x=3, y=1.5, style="r4/1.5", color="dodgerblue", pen="2p,black")
```